crates/bevy_ui/src/ui_node.rs

```diff
@@ -78,6 +78,9 @@ impl Default for Node {
     }
 }
 
+#[derive(Component, Copy, Clone, Debug, Default)]
+pub struct UiStackIndex(pub u32);
```

Comment:
Does it make sense to make `.0` pub? Users might expect that setting it does something but it's set unconditionally in `ui_stack_system`.

Reply:
Yep, this was meant to be private with an accessor function, I'll change it.